llvm/test/CodeGen/RISCV/zibimm.ll

```diff
@@ -0,0 +1,299 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=riscv32 -mattr=+experimental-zibimm -verify-machineinstrs < %s \
+; RUN:     | FileCheck -check-prefix=RV32I-ZIBIMM %s
```

Comment:
Use a common check prefix for RV32 and RV64 since the code is identical for all cases.